src/fairchem/core/common/distutils.py

```diff
@@ -15,11 +15,12 @@
 
 import torch
 import torch.distributed as dist
+from torch.distributed.elastic.utils.distributed import get_free_port
 
 from fairchem.core.common.typing import none_throws
 
 T = TypeVar("T")
-
+DISTRIBUTED_PORT = 13356
```

Comment:
Do we anticipate anyone needing to change this? Do we still want that flexibility as a command line argument?

Reply:
i dont think so, this shouldnt be user controlled, we shouldnt need this here but didnt want to mess with it too much